src/test_edgepi/integration_tests/test_digital_out/test_digital_out.py

```diff
@@ -34,3 +36,39 @@ def test_output_low(pin_name):
     dout=EdgePiDigitalOutput()
     dout.digital_output_state(pin_name, False)
     time.sleep(1)
+    gpio_stat, _ = dout.get_state(pin_name)
+    assert gpio_stat is False
+
+@pytest.mark.parametrize("pin_name", [
+    (GpioPins.DOUT1),
+    (GpioPins.DOUT2),
+    (GpioPins.DOUT3),
+    (GpioPins.DOUT4),
+    (GpioPins.DOUT5),
+    (GpioPins.DOUT6),
+    (GpioPins.DOUT7),
+    (GpioPins.DOUT8),
+])
+def test_direction_in(pin_name):
+    dout=EdgePiDigitalOutput()
+    dout.digital_output_direction(pin_name, True)
+    time.sleep(1)
```

Comment:
what is this sleep for?

Reply:
it was there to check the LED response when DOUT state changes